docs/compare.rst

```diff
@@ -1,35 +1,28 @@
 Mock Library Comparison
 =======================
 
-(flexmock for Mox or Mock users.)
+(flexmock for Mock users.)
 ---------------------------------------------------------------
 
 This document shows a side-by-side comparison of how to accomplish some
-basic tasks with flexmock as well as other popular Python mocking libraries.
+basic tasks with flexmock compared to Python unittest Mock.
 
 Simple fake object (attributes only)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
     # flexmock
-    mock = flexmock(some_attribute="value", some_other_attribute="value2")
-    assertEquals("value", mock.some_attribute)
-    assertEquals("value2", mock.some_other_attribute)
-
-    # Mox
-    mock = mox.MockAnything()
-    mock.some_attribute = "value"
-    mock.some_other_attribute = "value2"
-    assertEquals("value", mock.some_attribute)
-    assertEquals("value2", mock.some_other_attribute)
+    my_mock = flexmock(some_attribute="value", some_other_attribute="value2")
+    assert my_mock.some_attribute == "value"
+    assert my_mock.some_other_attribute == "value2"
 
     # Mock
     my_mock = mock.Mock()
     my_mock.some_attribute = "value"
     my_mock.some_other_attribute = "value2"
-    assertEquals("value", my_mock.some_attribute)
-    assertEquals("value2", my_mock.some_other_attribute)
+    assert my_mock.some_attribute == "value"
+    assert my_mock.some_other_attribute == "value2"
```

Comment:
`unittest.Mock` itself doesn't affect the formatting. It is the test runner that can change the formatting. Pytest formats assertions nicely but Python stdlib unittest doesn't.

I think plain `assert` is better in these examples since `assertEqual` is actually a method of `unittest.TestCase` and not a plain function.